### PR TITLE
Add ESPEasy to the vendor list

### PR DESCRIPTION
**Pull request by TD-er**
#### Summary

ESPEasy is an OpenSource firmware for ESP8266/ESP8285/ESP32 microcontrollers.
This firmware has support for over 100 sensors via so called plugins.
An ESPEasy setup can handle several tasks which create an instance of such plugin to collect data.
This data can then be sent to various controllers, like HTTP, MQTT, etc. 
One of these controllers is a controller to send to TheThingsNetwork using a Microchip RN2483 LoRaWAN module.
Support for other modules will be added in the future.

For this, a decoder is written to support data from all plugins.
By adding ESPEasy to the lorawan-devices selection, users of ESPEasy may experience an even simpler configuration to get started with TTN/LoRaWAN.

#### Changes
Added:
- ESPEasy to the vendor list
- Decoder JavaScript
- Profiles for various regions around the world, split into RN2483/RN2903 and Microchip firmware revision
- Some photos

#### Notes for Reviewers
The Heltec regional profiles were used and slightly modified for this configuration.

#### Release Notes
Added support for ESPEasy using Microchip RN2xx3 LoRaWAN module.

Closes: https://github.com/letscontrolit/ESPEasy/issues/3613

### CODE REVIEW

File: vendor/index.yaml
Comment (Jaime-Trinidad) [inline vendor/index.yaml:1653]: Hello @TD-er please take out commented lines, to have the same format as other device makers. Thank you

Comment (TD-er) [reply vendor/index.yaml]: Done.
I also removed it from another entry a few lines above.

**Review by Jaime-Trinidad — COMMENTED** [review]
Let's wait for a final review, files looks complete, index changes are already done

**Review by johanstokking — APPROVED** [review]

**Comment by TD-er** [comment]
Just curious, how long does it take for the entry to show up  on the console to add a new device?

**Review by Jaime-Trinidad — APPROVED** [review]
Files look complete

**Comment by johanstokking** [comment]
@TD-er on the next release of The Things Stack. That's gonna be 3.13.0. See https://github.com/TheThingsNetwork/lorawan-stack/milestones?direction=asc&sort=due_date for the planning.

We're tracking an internal issue to automatically update the device repository, but as we're on a biweekly release schedule we don't find that really necessary.

Refs https://github.com/TheThingsIndustries/lorawan-stack-aws/issues/352

**Comment by TD-er** [comment]
OK, then I have to wait till then to add screenshots and other descriptions to our documentation.

**Pull request merged**
